### DISCUSSION TITLE
Unable to open Github.com ..//This site can’t be reached

### DISCUSSION THREAD

**Discussion by JHONNYCODER** [community/community#74982] [Other Feature Feedback, Questions, & Ideas] [answered]
### Issue regarding Proxy or??

Question

### Body

I am unable to open Github.com on my pc I tried changing my Wifi but using same wifi on my smartphone and it workd there but on my pc on any other network it doesn't open also used other browsers still same thing happens.

It just shows .

This site can’t be reachedThe webpage at https://github.com/topics/ might be temporarily down or it may have moved permanently to a new web address.
**ERR_ADDRESS_INVALID**

I had to use VPN to access it tried resetting proxies and changing them disabling windows defender etc but didn't worked at all . Please help me.

**Comment by blakealmon** [comment, marked as answer]
Possible problems
(1) you installed a virus which manually changed your ability to visit github. i recommend you download malwarebytes or another virus scanner and run a full scan on your pc.

(2) Your blocked Google Chrome Helper with your firewall. just unblock it in your firewall

(3) you might need to manually change the ability to access websites yourself if you downloaded a virus and here is a link to a video to help [https://www.youtube.com/watch?v=wR9QAMLUAa4&ab_channel=TechTech](url)

**Reply by JHONNYCODER** [reply]
Thanks mate the last youtube video you mentioned in the 3rd point helped me ... Some virus may have edited my host file and thus this problem arised..
